From e0c78d59082aaadb95fc301af239c991686a1822 Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dave.stevenson@raspberrypi.com>
Date: Wed, 8 May 2024 18:05:08 +0100
Subject: [PATCH 1085/1085] dts: bcm2712: Drop snd_bcm2835 bootargs references
 from Pi5

Pi5 has no VCHIQ to support the snd_bcm2835 firmware audio
driver, so remove the reference to it from bootargs.

https://forums.raspberrypi.com/viewtopic.php?p=2219395#p2219395

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.com>
---
 arch/arm/boot/dts/broadcom/bcm2712-rpi-5-b.dts  | 2 +-
 arch/arm/boot/dts/broadcom/bcm2712-rpi-cm5.dtsi | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

--- a/arch/arm/boot/dts/broadcom/bcm2712-rpi-5-b.dts
+++ b/arch/arm/boot/dts/broadcom/bcm2712-rpi-5-b.dts
@@ -426,7 +426,7 @@ dpi_16bit_gpio2:        &rp1_dpi_16bit_g
 
 / {
 	chosen: chosen {
-		bootargs = "reboot=w coherent_pool=1M 8250.nr_uarts=1 pci=pcie_bus_safe snd_bcm2835.enable_compat_alsa=0 snd_bcm2835.enable_hdmi=1";
+		bootargs = "reboot=w coherent_pool=1M 8250.nr_uarts=1 pci=pcie_bus_safe";
 		stdout-path = "serial10:115200n8";
 	};
 
--- a/arch/arm/boot/dts/broadcom/bcm2712-rpi-cm5.dtsi
+++ b/arch/arm/boot/dts/broadcom/bcm2712-rpi-cm5.dtsi
@@ -426,7 +426,7 @@ dpi_16bit_gpio2:        &rp1_dpi_16bit_g
 
 / {
 	chosen: chosen {
-		bootargs = "reboot=w coherent_pool=1M 8250.nr_uarts=1 pci=pcie_bus_safe snd_bcm2835.enable_compat_alsa=0 snd_bcm2835.enable_hdmi=1";
+		bootargs = "reboot=w coherent_pool=1M 8250.nr_uarts=1 pci=pcie_bus_safe";
 		stdout-path = "serial10:115200n8";
 	};
 
